src/components/navigation-mobile/navigation-mobile.component.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Link } from "react-router-dom";
+// Material UI
+import { makeStyles } from "@material-ui/core/styles";
+import BottomNavigation from "@material-ui/core/BottomNavigation";
+import BottomNavigationAction from "@material-ui/core/BottomNavigationAction";
+// SVG's
+import { ReactComponent as ProjectsIcon } from "../../assets/icons/projects.svg";
+import { ReactComponent as BulletJournalIcon } from "../../assets/icons/journal.svg";
+import { ReactComponent as Logo } from "../../assets/icons/sloth.svg";
+import { ReactComponent as ProfileIcon } from "../../assets/icons/profile.svg";
+import { ReactComponent as GuestIcon } from "../../assets/icons/guest.svg";
+
+const useStyles = makeStyles({
+    root: {
+        width: "100vw",
+        position: "fixed",
+        bottom: 0,
+        left: 0,
+    },
+    icons: {
+        width: "4vh",
+        height: "4vh",
+    },
+});
+
+export default function NavigationMobile() {
+    const classes = useStyles();
+    const [value, setValue] = React.useState(0);
+    // Tests
+    const isSignedIn = true;
+    return (
+        <BottomNavigation
+            value={value}
+            onChange={(event, newValue) => {
+                setValue(newValue);
+            }}
+            showLabels
+            className={classes.root}
+        >
+            <BottomNavigationAction
+                component={Link}
+                to="/"
+                label="Home"
+                icon={<Logo className={classes.icons} />}
+            />
+            <BottomNavigationAction
+                component={Link}
+                to="/projects"
+                label="Projects"
+                icon={<ProjectsIcon className={classes.icons} />}
```

Comment:
Maybe not a thing to wrap your head around for now, but maybe in the future you should create some unified dynamic Icon component that changes based on arguments you send him. I believe that our app will use a lot of different icons

Reply:
Sure, will keep that in mind.